### ISSUE TITLE
Post type and Featured Image options disappeared

### ISSUE THREAD

**Issue by LordRembo** [WordPress/gutenberg#66637]
### Description

I'm on Wordpress 6.6.2 and during one of the recent updates, the 'post formats' as well as 'featured image' options have disappeared on blog posts as well as any custom post type I had.
I've disabled plugins (as I heard that Jetpack sometimes messes with admin stuff) but didn't make a difference.
Did something change with regards to enabling/disabling these 2 features? 

### Step-by-step reproduction instructions

1. Go to any post (new or edit existing)
2. look in the sidebar or open preferences
3. There's no option to add a Featured Image, nor is there a list of post types (just a default one)

### Environment info

- Wordpress 6.6.2
- Any browser

### Please confirm that you have searched existing issues in the repo.

- [x] Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

- [x] Yes

### Please confirm which theme type you used for testing.

- [ ] Block
- [ ] Classic
- [ ] Hybrid (e.g. classic with theme.json)
- [x] Not sure

**Comment by carolinan** [comment]
Hi, did you perhaps change your theme? All themes do not support post formats.

This is a screenshot of the post sidebar in 6.5.5. The post format is a select list, and the featured image is in its own panel.
![Image](https://github.com/user-attachments/assets/5fe98b7d-f66e-4ed8-93ba-419625771727)

This is a screenshot of the post sidebar in 6.6.2. **The design changes are intentional, and not a bug.**
The featured image option is at the top below the post title, and the format option is at the bottom, below the discussion settings.
![Image](https://github.com/user-attachments/assets/038a3cbf-d6fc-4ee1-a656-bfea6b117e4c)

If it is related to your custom post type, it will be difficult for other contributors to test, are you able to add more information about how you have added the custom post type?

**Comment by LordRembo** [comment]
@carolinan My sidebar looks identical, apart from the fact that 'Set featured image' is not in there. And clicking on the 'Standard' format just opens a modal with 1 option instead of all of them.
It's not related to the custom post type, as the same issue happens on the 'normal' posts as well. I didn't make any changes to my theme recently and everything worked perfectly fine the last time I made a post (in September). So it must be related to a Wordpress update.
Was some new option added or a setting changed in recent updates?

**Comment by carolinan** [comment]
Just like post formats, themes must register support for the featured image, otherwise the option will not be available.
What happens if you switch to one of the default themes like Twenty Twenty-Four?

These pages links to lists of all changes in 6.6.2 and 6.6.1.
https://wordpress.org/documentation/wordpress-version/version-6-6-2/
https://wordpress.org/documentation/wordpress-version/version-6-6-1/

**Comment by LordRembo** [comment]
@carolinan The option is registered in the functions.php: ` add_theme_support( 'post-thumbnails' );` This worked just fine before version 6.6.2, which released in September, AFTER I last added content, when featured images still worked.
I looked at the changelog & the changes in Trac, but nothing in there means anything to me to be honest.

**Comment by LordRembo** [comment]
@carolinan Okay, so I accidentally fixed it. This is the weirdest thing… I was looking through information on the 6.x release and saw that now there are 'Block' themes as well as 'Classic' themes, so I looked up if there were any changes to the basic structure of themes.
Turns out that themes are now expecting the `style.css` file (which contains the theme name and version) to be in the theme's root. 
I just added an empty CSS file with the theme name, opened up a post Edit and voilà, featured images option as well as Post formats are visible again.
I've had it in a `css` folder for 8 years without a problem so it's weird that this requirement (which is only needed to fetch the theme name I think?) suddenly is causing issues on a minor update.
